docs/fsharp/language-reference/plaintext-formatting.md

```diff
@@ -0,0 +1,339 @@
+---
+title: "Tutorial: Plain Text Formatting"
+description: Learn how to use structured formatted plain text in F# applications and scripts.
+ms.date: 07/22/2020
+---
+
+# Plain Text Formatting
+
+F# allows values to be formatted as structured plain text in a compositional way.
+As a simple example, consider the following and note how the otuput has been formatted as a matrix-like display of tuples.
+
+```console
+dotnet fsi
+
+> let data = [ for i in 1 .. 3 -> [ for j in 1 .. 3 -> (i, j) ] ];;
+
+val data : (int * int) list list =
+  [[(1, 1); (1, 2); (1, 3); (1, 4); (1, 5)];
+   [(2, 1); (2, 2); (2, 3); (2, 4); (2, 5)];
+   [(3, 1); (3, 2); (3, 3); (3, 4); (3, 5)];
+   [(4, 1); (4, 2); (4, 3); (4, 4); (4, 5)];
+   [(5, 1); (5, 2); (5, 3); (5, 4); (5, 5)]]
+
+>  printfn "%A" data;;
+[[(1, 1); (1, 2); (1, 3); (1, 4); (1, 5)];
+ [(2, 1); (2, 2); (2, 3); (2, 4); (2, 5)];
+ [(3, 1); (3, 2); (3, 3); (3, 4); (3, 5)];
+ [(4, 1); (4, 2); (4, 3); (4, 4); (4, 5)];
+ [(5, 1); (5, 2); (5, 3); (5, 4); (5, 5)]]
+```
+
+Plain text formatting is activated when using the `%A` format in printf formatting strings. This has limited customizability.
```

Comment:
```suggestion
Plain text formatting is activated when you use the `%A` format in `printf` formatting strings.
```
Eliding the bit about customization since it doesn't really matter here